src/state_machine.zig

```diff
@@ -995,6 +1144,26 @@ pub fn StateMachineType(comptime Storage: type, comptime constants_: struct {
                         .user_data = .{ .commit_entries_max = batch_accounts_max },
                         .ledger = .{ .commit_entries_max = batch_accounts_max },
                         .code = .{ .commit_entries_max = batch_accounts_max },
+                    },
+                },
+                .accounts_mutable = .{
+                    .prefetch_entries_max = std.math.max(
+                        // create_account()/lookup_account() looks up 1 AccountMutable per item.
+                        batch_accounts_max,
+                        // create_transfer()/post_or_void_pending_transfer() looks up 2
+                        // AccountMutables for every transfer.
+                        2 * batch_transfers_max,
+                    ),
+                    .tree_options_object = .{
+                        .cache_entries_max = options.cache_entries_accounts,
+                        .commit_entries_max = math.max(
+                            batch_accounts_max,
+                            // ×2 because creating a transfer will update 2 AccountsMutable.
```

Comment:
```suggestion
                            // ×2 because creating a transfer will update 2 AccountMutable's.
```

since we are referring to the item, not the groove.